src/module_EARTH_GRID_COMP.F90

```diff
@@ -334,6 +356,17 @@ SUBROUTINE EARTH_REGISTER(EARTH_GRID_COMP,RC_REG)
           file=__FILE__)) &
           return  ! bail out
       endif
+      if (.not.NUOPC_FieldDictionaryHasEntry( &
+        "mean_sensi_heat_flx_atm")) then
```

Comment:
Denise, we do not have this change on "mean_sensi_heat_flx_atm" and "mean_laten_heat_flx_atm" in fv3atm, do we need  to change them in fv3 too?